website/content/partials/known-issues/internal-error-namespace-missing-policy.mdx

```diff
@@ -9,4 +9,53 @@ This impacts all auth methods.
 - 1.14.4 and 1.14.5
 - 1.15.0 and 1.15.1
 
-A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
\ No newline at end of file
+A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
+
+### Workaround
+During authentication, Vault will derive any additional policies that should be inherited based on any groups an entity is a member of. If Vault attempts to attach a policy to a token from a different namespace than the one handling the authentication, _and_ that policy does not exist, Vault will return an internal error. The way to resolve this issue is to either add the policy, or to delete the group policy mapping referring to the non-existent policy.
+
+Below is an example of a userpass auth method that is failing because of a
+missing group policy from another namespace.
+
+```
+# Failed login
+$ vault login -method=userpass username=user1 password=123
+Error authenticating: Error making API request.
+
+URL: PUT http://127.0.0.1:8200/v1/auth/userpass/login/user1
+Code: 500. Errors:
+
+* internal error
+
+# Identify the entity and group IDs
+$ vault read -format=json identity/entity/name/user1 | jq '{"entity_id": .data.id, "group_ids": .data.group_ids} '
+{
+  "entity_id": "420c82de-57c3-df2e-2ef6-0690073b1636",
+  "group_ids": [
+    "6cb152b7-955d-272b-4dcf-a2ed668ca1ea"
+  ]
+}
+
```

Comment:
````suggestion
To confirm the problem is a missing policy, start by identifying the relevant
entity and group IDs:

<CodeBlockConfig hideClipboard>

```shell-session
$ vault read -format=json identity/entity/name/user1 | \
  jq '{"entity_id": .data.id, "group_ids": .data.group_ids} '
{
  "entity_id": "420c82de-57c3-df2e-2ef6-0690073b1636",
  "group_ids": [
    "6cb152b7-955d-272b-4dcf-a2ed668ca1ea"
  ]
}
```

</CodeBlockConfig>

````

Style correction: avoid multiple steps per code block, introduce what the example code is doing